app/models/ticket_purchase.rb

```diff
@@ -60,6 +63,13 @@ def self.update_quantity(conference, quantity, ticket, user)
     purchase.quantity = quantity if quantity > 0
     purchase
   end
+
+  def pay(payment)
+    update_attributes(paid: true, payment: payment)
+    PhysicalTicket.transaction do
```

Comment:
Why do we need to wrap this into a transaction? What the difference between creating one of them or any of them?

I am not pretty sure how we should handle this. @mdeniz opinions/ideas?